src/main/java/com/spiralgo/ecommserver/resource/TagResource.java

```diff
@@ -27,4 +30,9 @@ public TagResource(ProductService productService) {
     public List<Tag> getProductsWithPageable() {
         return productService.getTags();
     }
+
+    @GetMapping(value = "/productsWith/{name}")
```

Comment:
```suggestion
    @GetMapping(value = "/productswith/{name}")
```

A minor issue but we can keep the URL style.

Reply:
I think "products-with" is more correct than both solutions.